worker_voucher/apps.py

```diff
@@ -1,6 +1,33 @@
 from django.apps import AppConfig
 
+DEFAULT_CONFIG = {
+    "gql_worker_voucher_search_perms": ["204001"],
+    "gql_worker_voucher_create_perms": ["204002"],
+    "gql_worker_voucher_update_perms": ["204003"],
+    "gql_worker_voucher_delete_perms": ["204004"],
+}
```

Comment:
Are those new perms? If so please remeber to include them on confluence page.

Reply:
I've added them already